### PR TITLE
Removing EventType dependency from eventing-contrib

### PR DESCRIPTION
**Pull request by nachocano**
Helps with https://github.com/knative/eventing/issues/2484

## Proposed Changes

- 🧽Removing runtime dependency on eventing due to EventType.
- 🧽Lot's of cleanups to the different sources.

Disclaimer: only tested the GithubSource.

See eventing PR https://github.com/knative/eventing/pull/2770 for more context

**Release Note**

```release-note
Sources in eventing-contrib no longer need EventType from eventing. 
```

**Docs**

/assign @lionelvillard @vaikas @n3wscott @mattmoor

### CODE REVIEW

File: awssqs/pkg/reconciler/awssqssource.go [hunk 1]
```diff
@@ -18,7 +18,6 @@ package reconciler
 
 import (
 	"context"
```
Comment (mattmoor) [inline awssqs/pkg/reconciler/awssqssource.go]: Format Go code:
```suggestion
	"context"

```

File: github/cmd/controller/main.go [hunk 2]
```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package main
 
 import (
-	github "knative.dev/eventing-contrib/github/pkg/reconciler"
+	"knative.dev/eventing-contrib/github/pkg/reconciler"
```
Comment (mattmoor) [inline github/cmd/controller/main.go]: Format Go code:
```suggestion
	github "knative.dev/eventing-contrib/github/pkg/reconciler"
```

File: github/pkg/reconciler/githubsource.go [hunk 3]
```diff
@@ -21,18 +21,13 @@ import (
...
 	"strings"
 
-	"go.uber.org/zap"
-
 	//k8s.io imports
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/equality"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
```
Comment (mattmoor) [inline github/pkg/reconciler/githubsource.go]: Format Go code:
```suggestion
	"k8s.io/apimachinery/pkg/labels"

```

File: github/pkg/reconciler/githubsource.go [hunk 4]
```diff
@@ -21,18 +21,13 @@ import (
...
-	eventingv1alpha1 "knative.dev/eventing/pkg/apis/eventing/v1alpha1"
-
 	//knative.dev/serving imports
-	v1 "knative.dev/serving/pkg/apis/serving/v1"
+	"knative.dev/serving/pkg/apis/serving/v1"
```
Comment (mattmoor) [inline github/pkg/reconciler/githubsource.go]: Format Go code:
```suggestion
	v1 "knative.dev/serving/pkg/apis/serving/v1"
```

File: awssqs/pkg/reconciler/awssqssource.go (hunk 1 above)
Comment (mattmoor) [inline awssqs/pkg/reconciler/awssqssource.go]: Format Go code:
```suggestion
	"context"

```

File: github/cmd/controller/main.go (hunk 2 above)
Comment (mattmoor) [inline github/cmd/controller/main.go]: Format Go code:
```suggestion
	github "knative.dev/eventing-contrib/github/pkg/reconciler"
```

File: github/pkg/reconciler/githubsource.go (hunk 3 above)
Comment (mattmoor) [inline github/pkg/reconciler/githubsource.go]: Format Go code:
```suggestion
	"k8s.io/apimachinery/pkg/labels"

```

File: github/pkg/reconciler/githubsource.go (hunk 4 above)
Comment (mattmoor) [inline github/pkg/reconciler/githubsource.go]: Format Go code:
```suggestion
	v1 "knative.dev/serving/pkg/apis/serving/v1"
```

File: awssqs/pkg/reconciler/awssqssource.go (hunk 1 above)
Comment (mattmoor) [inline awssqs/pkg/reconciler/awssqssource.go]: Format Go code:
```suggestion
	"context"

```

File: github/cmd/controller/main.go (hunk 2 above)
Comment (mattmoor) [inline github/cmd/controller/main.go]: Format Go code:
```suggestion
	github "knative.dev/eventing-contrib/github/pkg/reconciler"
```

File: github/pkg/reconciler/githubsource.go (hunk 3 above)
Comment (mattmoor) [inline github/pkg/reconciler/githubsource.go]: Format Go code:
```suggestion
	"k8s.io/apimachinery/pkg/labels"

```

File: github/pkg/reconciler/githubsource.go (hunk 4 above)
Comment (mattmoor) [inline github/pkg/reconciler/githubsource.go]: Format Go code:
```suggestion
	v1 "knative.dev/serving/pkg/apis/serving/v1"
```

File: awssqs/pkg/reconciler/awssqssource.go (hunk 1 above)
Comment (mattmoor) [inline awssqs/pkg/reconciler/awssqssource.go]: Format Go code:
```suggestion
	"context"

```

File: github/cmd/controller/main.go (hunk 2 above)
Comment (mattmoor) [inline github/cmd/controller/main.go]: Format Go code:
```suggestion
	github "knative.dev/eventing-contrib/github/pkg/reconciler"
```

File: github/pkg/reconciler/githubsource.go (hunk 3 above)
Comment (mattmoor) [inline github/pkg/reconciler/githubsource.go]: Format Go code:
```suggestion
	"k8s.io/apimachinery/pkg/labels"

```

File: github/pkg/reconciler/githubsource.go (hunk 4 above)
Comment (mattmoor) [inline github/pkg/reconciler/githubsource.go]: Format Go code:
```suggestion
	v1 "knative.dev/serving/pkg/apis/serving/v1"
```

**Comment by n3wscott** [comment]
omg 👍 🥇

File: awssqs/pkg/reconciler/awssqssource.go (hunk 1 above)
Comment (mattmoor) [inline awssqs/pkg/reconciler/awssqssource.go]: Format Go code:
```suggestion
	"context"

```

File: github/cmd/controller/main.go (hunk 2 above)
Comment (mattmoor) [inline github/cmd/controller/main.go]: Format Go code:
```suggestion
	github "knative.dev/eventing-contrib/github/pkg/reconciler"
```

File: github/pkg/reconciler/githubsource.go (hunk 3 above)
Comment (mattmoor) [inline github/pkg/reconciler/githubsource.go]: Format Go code:
```suggestion
	"k8s.io/apimachinery/pkg/labels"

```

File: github/pkg/reconciler/githubsource.go (hunk 4 above)
Comment (mattmoor) [inline github/pkg/reconciler/githubsource.go]: Format Go code:
```suggestion
	v1 "knative.dev/serving/pkg/apis/serving/v1"
```

File: awssqs/pkg/reconciler/awssqssource.go (hunk 1 above)
Comment (mattmoor) [inline awssqs/pkg/reconciler/awssqssource.go]: Format Go code:
```suggestion
	"context"

```

File: github/cmd/controller/main.go (hunk 2 above)
Comment (mattmoor) [inline github/cmd/controller/main.go]: Format Go code:
```suggestion
	github "knative.dev/eventing-contrib/github/pkg/reconciler"
```

File: github/pkg/reconciler/githubsource.go (hunk 3 above)
Comment (mattmoor) [inline github/pkg/reconciler/githubsource.go]: Format Go code:
```suggestion
	"k8s.io/apimachinery/pkg/labels"

```

File: github/pkg/reconciler/githubsource.go (hunk 4 above)
Comment (mattmoor) [inline github/pkg/reconciler/githubsource.go]: Format Go code:
```suggestion
	v1 "knative.dev/serving/pkg/apis/serving/v1"
```

File: awssqs/pkg/reconciler/awssqssource.go (hunk 1 above)
Comment (mattmoor) [inline awssqs/pkg/reconciler/awssqssource.go]: Format Go code:
```suggestion
	"context"

```

File: github/cmd/controller/main.go (hunk 2 above)
Comment (mattmoor) [inline github/cmd/controller/main.go]: Format Go code:
```suggestion
	github "knative.dev/eventing-contrib/github/pkg/reconciler"
```

File: github/pkg/reconciler/githubsource.go (hunk 3 above)
Comment (mattmoor) [inline github/pkg/reconciler/githubsource.go]: Format Go code:
```suggestion
	"k8s.io/apimachinery/pkg/labels"

```

File: github/pkg/reconciler/githubsource.go (hunk 4 above)
Comment (mattmoor) [inline github/pkg/reconciler/githubsource.go]: Format Go code:
```suggestion
	v1 "knative.dev/serving/pkg/apis/serving/v1"
```

File: awssqs/pkg/reconciler/awssqssource.go (hunk 1 above)
Comment (nachocano) [reply awssqs/pkg/reconciler/awssqssource.go]: done

File: github/cmd/controller/main.go (hunk 2 above)
Comment (nachocano) [reply github/cmd/controller/main.go]: done

File: github/pkg/reconciler/githubsource.go (hunk 3 above)
Comment (nachocano) [reply github/pkg/reconciler/githubsource.go]: done

File: github/pkg/reconciler/githubsource.go (hunk 4 above)
Comment (nachocano) [reply github/pkg/reconciler/githubsource.go]: done

**Comment by nachocano** [comment]
@slinkydeveloper it seems that moving to v2 broker some UTs in kafka? The message does not have Key anymore? Can you check?

afka/source/pkg/adapter/message.go:102:8: msg.Key undefined (type *kafka_sarama.Message has no field or method Key)

**Comment by slinkydeveloper** [comment]
@nachocano https://github.com/knative/eventing-contrib/pull/927

**Comment by nachocano** [comment]
@n3wscott this should be ready to go in if the integration test pass...
Can you take another look?

**Pull request merged**